composer.json

```
@@ -30,20 +30,20 @@
         "doctrine/persistence": "^2.4",
         "phpcr/phpcr": "^2.1.1",
         "phpcr/phpcr-implementation": "^2.1",
-        "phpcr/phpcr-utils": "^1.3.0",
+        "phpcr/phpcr-utils": "^1.3.0 || ^2.0@dev",
```

when phpcr-utils 2.0 is released:

```suggestion
        "phpcr/phpcr-utils": "^1.3.0 || ^2.0",
```